pkg/karmadactl/cmdinit/kubernetes/deploy.go

```diff
@@ -613,6 +615,19 @@ func (i *CommandInitOption) karmadaAggregatedAPIServerImage() string {
 	return i.KarmadaAggregatedAPIServerImage
 }
 
+// get image pull secret
+func (i *CommandInitOption) getImagepullsecrets() []corev1.LocalObjectReference {
```

Comment:
```suggestion
func (i *CommandInitOption) getImagePullSecrets() []corev1.LocalObjectReference {
```
It seems clearer for camel case.